lib/msal-node/src/error/NodeAuthError.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AuthError } from "@azure/msal-common";
+
+/**
+ * NodeAuthErrorMessage class containing string constants used by error codes and messages.
+ */
+export const NodeAuthErrorMessage = {
+    stateNotFoundError: {
+        code: "state_not_found",
+        desc: "State not found. Please verify that the request originated from msal."
+    },
+   
+    unableToParseStateError: {
```

Comment:
Is this being used anywhere?

Reply:
+1

Reply:
In validateState method in ClientApplication.ts

Reply:
I see the `stateNotFoundError` in ClientApplication, but not this one?

Reply:
I am actually not using unableToParseStateError, instead, I am using StateMismatchError from msal common.

Reply:
If this isn't being used anywhere, please remove.